src/components/ButtonPanel.js

```diff
@@ -1,26 +1,30 @@
-import React from 'react';
-import {Button} from './Button';
+import React from "react";
+import {Button} from "./Button";
 
 export const ButtonPanel = (props) => {
+
+  const handleClick = (buttonName) => props.clickHandler(buttonName);
   
   const renderButtons = () => {
     let buttons;
     const groups = [
-      ['AC', '+/-', '%', '+'],
-      ['7', '8', '9', 'X'],
-      ['4', '5', '6', '-'],
-      ['1', '2', '3', '+'],
-      ['0', '.', '=', null]
+      ["AC", "+/-", "%", "+"],
+      ["7", "8", "9", "X"],
+      ["4", "5", "6", "-"],
+      ["1", "2", "3", "/"],
+      ["0", ".", "=", null]
     ]
 
     buttons = groups.map((group) => {
       let [a,b,c,d] = group;
-      return (<div>
-          <Button value = {a} className = 'button'/>
-          <Button value = {b} className = 'button'/>
-          <Button value = {c} className = {d ? 'button' : 'button orange-btn'} />
-          {d && <Button value = {d} className = 'button orange-btn'/>}
-        </div>)
+      return (
+        <div key={groups.indexOf(group)}>
+          <Button value={a} wide={a === '0' ? true : false} backgroundColor = "#EEE" onClick={handleClick}/>
+          <Button value={b} backgroundColor = "#EEE" onClick={handleClick}/>
+          <Button value={c} backgroundColor = {d && "#EEE"}  onClick={handleClick}/>
+          {d && <Button value={d} onClick={handleClick}/>}
+        </div>
```

Comment:
it's tricky/confusing sometimes to pass a prop called `onClick` lol, sometimes we confuse it with javascript's `"onClick"`

That's why this project proposes the name "`clickHandler`" [ [SCREENSHOT](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/passToButton.png) ]